tox.ini

```diff
@@ -9,11 +9,12 @@ setenv =
 deps =
     -r{toxinidir}/requirements.txt
     -r{toxinidir}/dev-requirements.txt
-    django18: Django>=1.8,<1.9
-    django111: Django>=1.11,<1.12
 
 # These are the equivalent of 'make ci'.
 commands =
+    django111: pip install 'Django>=1.11,<1.12'
+    py36-django20: pip install 'Django>=2.0,<2.1'
+    py36-django21: pip install 'Django>=2.1'
```

Comment:
One last minor point, may have just missed this before; this will implicitly switch to testing Django 2.2 pretty soon, probably want an upper version limit here.

Reply:
Right, thanks for catching this :-)